src/build.ts

```diff
@@ -274,17 +275,25 @@ async function _build(
   //   await symlink(resolve(ctx.rootDir), nodemodulesDir).catch(() => {})
   // }
 
-  // untyped
-  await typesBuild(ctx);
-
-  // mkdist
-  await mkdistBuild(ctx);
-
-  // rollup
-  await rollupBuild(ctx);
-
-  // copy
-  await copyBuild(ctx);
+  const buildTasks = [
+    // untyped
+    typesBuild(ctx),
+    // mkdist
+    mkdistBuild(ctx),
+    // rollup
+    rollupBuild(ctx),
+    // copy
+    copyBuild(ctx),
```

Comment:
This is almost correct however we need to do something like ()=>copybuild(ctx) here because otherwise regardless of awaiting or not, they will be started immediately in parallel

Reply:
Oh, I was careless—it's already fixed now.